.eslintrc

```diff
@@ -97,5 +97,8 @@
             ]
         }],
         "space-in-parens": "error"
+    },
```

Comment:
they are needed since we started to use BigInt which is supported in es2018. Also node version needs to be updated, bigint is supported 10.4 and above.

I think gitignore change is coming from another pr since I merged master into this pr a while ago